web-console/Storage/OB-OB-Lifecycle-Management/OB-OBC-Lifecycle.md

```diff
@@ -0,0 +1,112 @@
+# OB/OBC Lifecycle Management Design
+Support object bucket (OB) and object bucket (OBC) CRUD
```

Comment:
Should we add any description around when and why these sections will be shown in the console UI?

Reply:
@shirimordechay 

Let's add some Background information at the top to provide some context:

- Object buckets (OBs) and Object bucket claims (OBCs) are storage constructs pertaining to object storage much in the same way that Persistent Volumes (PVs) and Persistent Volume Claims (PVCs) are used to present storage constructs for block and file storage.
- The OBs and OBCs rely on native storage classes just as PVs and PVCs do.
- Having OBs and OBCs in the UI allows an end-user to be able to create claims against the object store just as one can do with PVs and PVCs today.  
- The OB/OBC functionality would be present in the UI just as PVs and PVCs are as long as the object storage provisioner is deployed.